### PR TITLE
feat: improve NftCardGrid

### PR DESCRIPTION
**Pull request by sk-umiuma**

### CODE REVIEW

**Comment by sk-umiuma** [comment]
https://user-images.githubusercontent.com/110230002/227431510-7dc849f2-1727-4a16-b34e-e28dfe28a16c.mp4

**Review by sk-yagi — APPROVED** [review]
LGTM!

File: containers/PageKolours/containers/NftCard/components/PaletteBar/index.tsx [hunk 1]
```diff
@@ -17,13 +17,18 @@ export default function PaletteBar({ className, style, value }: Props) {
...
     <div className={cx(styles.container, className)} style={style}>
       <Flex.Row className={styles.box}>
-        {value.map(({ kolour }, index) => (
-          <div
-            key={index}
-            className={styles.item}
-            style={{ backgroundColor: toHexColor(kolour) }}
-          />
-        ))}
+        <>
```
Comment (sk-kitsune) [inline containers/PageKolours/containers/NftCard/components/PaletteBar/index.tsx:20]: unnecessary `<></>`

File: containers/PageKolours/containers/NftCard/components/PaletteBar/index.tsx [hunk 2]
```diff
@@ -17,13 +17,18 @@ export default function PaletteBar({ className, style, value }: Props) {
   return (
     <div className={cx(styles.container, className)} style={style}>
       <Flex.Row className={styles.box}>
```
Comment (sk-kitsune) [inline containers/PageKolours/containers/NftCard/components/PaletteBar/index.tsx:19]: `div` should be sufficient.

`Flex.Box` is unnecessary

File: containers/PageKolours/containers/NftCardGrid/index.tsx [hunk 3]
```diff
@@ -29,18 +31,28 @@ export default function NftCardGrid({
...
+  /**
+   * Current order of each sort index:
+   * - true: descending
+   * - false: ascending
+   */
+  const [sortOrders, setSortOrders] = React.useState({
+    0: true,
+    1: true,
+    2: true,
+  });
```
Comment (sk-kitsune) [inline containers/PageKolours/containers/NftCardGrid/index.tsx:46]: `const [reversed, setReversed] = React.useState(false);`

File: containers/PageKolours/containers/NftCardGrid/index.tsx [hunk 4]
```diff
@@ -55,16 +67,57 @@ export default function NftCardGrid({
...
+              setSortOrders({
+                ...sortOrders,
+                0: sortIndex === 0 ? !sortOrders[0] : true,
+              });
+              setSortIndex(0);
+            }}
+            icon={<IconSort />}
+            iconPosition="right"
+          />
+          <Button
```
Comment (sk-kitsune) [inline containers/PageKolours/containers/NftCardGrid/index.tsx:93]: let's wait for the PR from @sk-gachou

File: modules/teiki-ui/components/Button/index.module.scss [hunk 5]
```diff
@@ -64,9 +62,7 @@
   }
 
   &:active {
-    background-color: #006e37;
-    border-color: transparent;
-    color: white;
```
Comment (sk-kitsune) [inline modules/teiki-ui/components/Button/index.module.scss:69]: revert these changes

File: containers/PageKolours/containers/NftCardGrid/index.tsx (hunk 4 above)
Comment (sk-kitsune) [reply containers/PageKolours/containers/NftCardGrid/index.tsx:93]: https://github.com/kreate-community/kreate-web/pull/199

Use this component
